core/src/test/scala/com/pingcap/tispark/auth/TiAuthIntegrationSuite.scala

```diff
@@ -164,8 +164,10 @@ class TiAuthIntegrationSuite extends SharedSQLContext {
   test(f"Show tables should not contain invisible table") {
     noException should be thrownBy spark.sql(s"use $databaseWithPrefix")
 
+    // spark 3.2 add isTemporary col when `show tables`, we need to exclude it.
     val tables = spark
       .sql(s"show tables")
+      .drop("isTemporary")
```

Comment:
Why should we exclude it, since this doesn't influence the test's result

Reply:
Sadly, it will influence the test's result. 
tables will contain (f"[$databaseWithPrefix,$table,flase]") rather than [$databaseWithPrefix,$table]
`false`  corresponds to the new column `isTemporary` in spark 3.2

Reply:
Got it, so should we mention it in README?

Reply:
it's the spark feature, I think we need not mention it for spark change.